### PR TITLE
Remove check for Rails 3

### PR DESCRIPTION
**Pull request by woodall**
This gem is alive and well in our Rails 4 app.

### CODE REVIEW

File: lib/basic_assumption.rb [hunk 1]
```diff
@@ -113,6 +113,6 @@ def define_writer_method(name)
   end
 end
 
-if defined?(Rails) && Rails::VERSION::MAJOR == 3
+if defined?(Rails)
```
Comment (mattyoho) [inline lib/basic_assumption.rb]: Just for a smaller change, what about switching to `if defined?(Rails) && Rails::VERSION::MAJOR >= 3`?

**Comment by mattyoho** [comment]
Rad! Happy to get something like this merged.

**Comment by woodall** [comment]
Good call. Updating now...

**Comment by woodall** [comment]
updated!

**Comment by mattyoho** [comment]
@woodall Thanks! :+1: Happy to merge this now. I can cut a new gem release later this evening.

**Comment by woodall** [comment]
I'd be happy to make the version change if that's all there is to it? If so, `0.5.2`?

**Comment by mattyoho** [comment]
@woodall Sure, and also toss a quick note into [HISTORY.rdoc](https://github.com/mattyoho/basic_assumption/blob/master/HISTORY.rdoc). Version `0.5.2` sounds good.

**Pull request merged**